galaxy.yml

```diff
@@ -8,5 +8,4 @@ name: nxos
 namespace: cisco
 readme: README.md
 repository: https://github.com/ansible-collections/nxos
-version: 0.0.1
```

Comment:
I think you need to retain the version key for galaxy? https://github.com/ansible-collections/ansible.netcommon/blob/ff06176c598d9b187722799dce3daaf911792c39/galaxy.yml#L10

Reply:
I brought this up in the meeting yesterday and I was told we can remove it. Also, nxos and netcommon seem to be the only collections that have this.